EXILED/Exiled.API/Features/Items/ExplosiveGrenade.cs

```diff
@@ -45,64 +40,30 @@ internal ExplosiveGrenade(ItemType type, Player player = null)
         {
         }
 
-        /// <summary>
-        /// Gets a <see cref="ExplosionGrenadeProjectile"/> to change grenade properties.
-        /// </summary>
-        public new ExplosionGrenadeProjectile Projectile { get; }
-
         /// <summary>
         /// Gets or sets the maximum radius of the grenade.
         /// </summary>
-        public float MaxRadius
-        {
-            get => Projectile.MaxRadius;
-            set => Projectile.MaxRadius = value;
-        }
+        public float MaxRadius { get; set; }
 
         /// <summary>
         /// Gets or sets the multiplier for damage against <see cref="Side.Scp"/> players.
         /// </summary>
-        public float ScpDamageMultiplier
-        {
-            get => Projectile.ScpDamageMultiplier;
-            set => Projectile.ScpDamageMultiplier = value;
-        }
+        public float ScpDamageMultiplier { get; set; }
 
         /// <summary>
         /// Gets or sets how long the <see cref="EffectType.Burned"/> effect will last.
         /// </summary>
-        public float BurnDuration
-        {
-            get => Projectile.BurnDuration;
-            set => Projectile.BurnDuration = value;
-        }
+        public float BurnDuration { get; set; }
 
         /// <summary>
         /// Gets or sets how long the <see cref="EffectType.Deafened"/> effect will last.
         /// </summary>
-        public float DeafenDuration
-        {
-            get => Projectile.DeafenDuration;
-            set => Projectile.DeafenDuration = value;
-        }
+        public float DeafenDuration { get; set; }
 
         /// <summary>
         /// Gets or sets how long the <see cref="EffectType.Concussed"/> effect will last.
         /// </summary>
-        public float ConcussDuration
-        {
-            get => Projectile.ConcussDuration;
-            set => Projectile.ConcussDuration = value;
-        }
-
-        /// <summary>
-        /// Gets or sets how long the fuse will last.
-        /// </summary>
-        public float FuseTime
-        {
-            get => Projectile.FuseTime;
-            set => Projectile.FuseTime = value;
-        }
+        public float ConcussDuration { get; set; }
 
```

Comment:
Why ? This change would be needed ?

Reply:
I originally didn't want to change the Projectile API structure, but still came to the conclusion that storing prefabs for throwables was a bad idea from the point of view of all cases😭
And removing it and writing all the properties and their relations correctly, is much more safer